src/Web/Server.hs

```diff
@@ -19,6 +19,7 @@ import Control.Monad.IO.Class
 import General.System
 import Control.Concurrent
 import Control.Exception
+import Data.Streaming.Network (HostPreference(..))
```

Comment:
This is exported from Network.Wai.Handler.Warp, which reduces the number of packages by 1.